scripts/build-boshrelease-pipelines.sh

```diff
@@ -48,3 +48,5 @@ setup_release_pipeline rds-broker alphagov/paas-rds-broker-boshrelease master
 setup_release_pipeline datadog-for-cloudfoundry alphagov/paas-datadog-for-cloudfoundry-boshrelease master
 setup_release_pipeline logsearch-for-cloudfoundry alphagov/paas-logsearch-for-cloudfoundry gds_master
 setup_release_pipeline paas-haproxy alphagov/paas-haproxy-release master
+setup_release_pipeline graphite CloudCredo/graphite-statsd-boshrelease master
```

Comment:
:+1: This will also mean that any PRs against upstream will get run on our CI. We specifically wanted to avoid running PRs from 3rd-parties on our CI at the moment.